napari/_qt/qt_main_window.py

```diff
@@ -488,6 +488,14 @@ def _add_file_menu(self):
         )
         screenshot_wv.triggered.connect(self._screenshot_dialog)
 
+        clipboard = QAction(
+            trans._('Copy Screenshot to Clipboard...'), self._qt_window
```

Comment:
The `...` are used to indicate that a new dialog will be opened, so we can remove it :)

Reply:
Thanks! Indeed it doesn't make sense for the `...` to be there.